src/Carbon/CarbonInterval.php

```diff
@@ -1706,4 +1706,116 @@ public function equalTo($interval): bool
 
         return $interval !== null && $this->totalMicroseconds === $interval->totalMicroseconds;
     }
+
+    /**
+     * Determines if the instance is greater (longer) than another
+     *
+     * @param \Carbon\CarbonInterval|DateInterval|mixed $interval
+     *
+     * @see greaterThan()
+     *
+     * @return bool
+     */
+    public function gt($interval): bool
+    {
+        return $this->greaterThan($interval);
+    }
+
+    /**
+     * Determines if the instance is greater (longer) than another
+     *
+     * @param \Carbon\CarbonInterval|DateInterval|mixed $interval
+     *
+     * @return bool
+     */
+    public function greaterThan($interval): bool
+    {
+        if (!($interval instanceof self)) {
+            $interval = self::make($interval);
+        }
+
+        return $interval !== null && $this->totalMicroseconds > $interval->totalMicroseconds;
```

Comment:
@burobo I would say if `$interval` is `null`, it's like 0 seconds, so `$this` is always "greater". For example `var_dump((new DateTime()) > null);` is `true` in PHP and `var_dump((new DateTime()) < null);`. It is also more consistent if `greaterThan` is `true` when `lessThan` is `false`.

Reply:
I get it. I'll fix it soon. Thank you!